packages/dx-react-scheduler/docs/reference/week-view.md

```diff
@@ -0,0 +1,169 @@
+# WeekView Plugin Reference
+
+A plugin that renders Scheduler data for a week. This plugin arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { WeekView } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { WeekView } from '@devexpress/dx-react-scheduler';
+```
+
+## User Reference
+
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+name? | string | `Week` | Specifies the view name. This is required property if you want to add many same plugins.
+excludedDays? | Array&lt;number&gt; | [] | Specifies days that is not displayed on the view.
```

Comment:
I think we can add the following note:
Sunday is 0, Monday is 1, and so on.

Reply:
Agree. Done.